packages/utils/src/constants/gas.ts

```diff
@@ -4,12 +4,26 @@ export const NATIVE_TOKEN = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
 export const DEFAULT_GAS_ESTIMATES = {
   execute: "400000",
   executeL1: "20000",
+  // average gas of reconcile tx to L2 / batch size (n=10)
+  // - 350,000 for 1 tx batch
+  // - 1,200,000 for 10 tx batch
+  proveAndProcess: "120000",
+  // average gas of reconcile tx L1 costs on op
+  // - 43,000 1 tx batch
+  // - 220,000 10 tx batch
+  proveAndProcessL1: "20000",
+  // monthly costs in weth / monthly transfer count = weth cost per transfer
+  // weth. chaindata should be updated when switching between op and slow mode
+  messaging: "4500",
```

Comment:
I am just wondering why we need `messaging` item here?

Does this only work in slow mode to go through AMB?